zipfile2/_zipfile.py

```diff
@@ -211,8 +211,7 @@ def _extract_member_to(self, member, arcname, targetpath, pwd,
             if not os.path.isdir(targetpath):
                 os.mkdir(targetpath)
             return targetpath
-
-        if is_zipinfo_symlink(member):
+        elif is_zipinfo_symlink(member):
```

Comment:
it looks that this function is not returning `targetpath` if this elif is executed